modrequest.go

```diff
@@ -17,4 +18,6 @@ func (b *bot) requestMod(m *discordgo.Message, parts []string) {
 		message = strings.Join(parts, " ")
 	}
 	b.discord.ChannelMessageSend(feedChannel, fmt.Sprintf(modRequest, m.Author.ID, m.ChannelID, url, message))
+	b.discord.ChannelMessageSend(m.ChannelID, fmt.Sprintf(requestRecv, m.Author.ID))
```

Comment:
I think you want to use `b.sendQuickPM(user, message)` instead of sending that message publicly in the channel?